README.md

```diff
@@ -0,0 +1,103 @@
+use cases on the [USAGE](https://github.com/GeoscienceAustralia/dea-notebooks/blob/stable/USAGE.rst)
+page.
+
+## Browse the DEA Knowledge Hub
+
+The Knowledge Hub is where you can easily view the notebooks and read DEA Tools documentation. You can browse the following sections.
+
+* **[Beginner’s guide](https://docs.dea.ga.gov.au/notebooks/Beginners_guide/)** &mdash;
+  Introductory notebooks aimed at introducing Jupyter Notebooks and how to load,
+  plot, and interact with DEA data.
+* **[DEA products](https://docs.dea.ga.gov.au/notebooks/DEA_products/)** &mdash;
+  Notebooks introducing DEA's satellite datasets and derived products,
+  including how to load each dataset and any special features of the data.
+* **[How-to guides](https://docs.dea.ga.gov.au/notebooks/How_to_guides/)** &mdash;
+  A recipe book of simple code examples demonstrating how to perform common
+  geospatial analysis tasks using DEA and open-source software.
+* **[Interactive apps](https://docs.dea.ga.gov.au/notebooks/Interactive_apps/)** &mdash;
+  Interactive apps and widgets that require little or no coding to run.
+* **[Real-world examples](https://docs.dea.ga.gov.au/notebooks/Real_world_examples/)** &mdash;
+  More complex case study workflows demonstrating how DEA can be
+  used to address real-world problems.
+* **[DEA Tools Package](https://docs.dea.ga.gov.au/notebooks/Tools/)** &mdash;
+  An open-source Python package containing functions and
+  algorithms to assist in analysing DEA data.
+
+## Run on the DEA Sandbox
+
+The **DEA Sandbox** is a free virtual environment for interacting with the notebooks and tools. You can use the sandbox to conduct geospatial analysis without installing anything on your computer. Get started with the [DEA Sandbox](https://app.sandbox.dea.ga.gov.au/).
+
+<a href="https://app.sandbox.dea.ga.gov.au/"><img src="Supplementary_data/DEA_Sandbox/DEA_Sandbox_cover_page.png" width="500" alt="The cover page of the DEA Sandbox." /></a>
+
+## Run on the NCI
+
+The National Computational Infrastructure is a supercomputer environment available to Australian researchers, academics, and other users by negotiation. The NCI can be used for high-performance geospatial analysis. Learn how to get started with the [NCI](https://docs.dea.ga.gov.au/guides/setup/NCI/README/).
+
+https://docs.dea.ga.gov.au/guides/setup/NCI/README/
+
+<span id="contribute"></span>
+
+## How to contribute
+
+To get started, see these articles in the wiki.
+
+* [Git workflow](https://github.com/GeoscienceAustralia/dea-notebooks/wiki/Git-workflow)
+* [Create a DEA Notebook](https://github.com/GeoscienceAustralia/dea-notebooks/wiki/Create-a-DEA-Notebook)
+* [Edit a DEA Notebook](https://github.com/GeoscienceAustralia/dea-notebooks/wiki/Edit-a-DEA-Notebook)
+
+We would also appreciate it if you could add your a citation to our
+[USAGE](https://github.com/GeoscienceAustralia/dea-notebooks/blob/stable/USAGE.rst)
+page.
+
+All notebooks in this repository contain tags describing
```

Comment:
I wonder if we should leave this paragraph out completely - the tags functionality currently needs a lot of work, and might not be something we want to highlight so prominently.